addOns/customreport/src/main/java/org/zaproxy/zap/extension/customreport/AlertDetailsPanel.java

```diff
@@ -100,6 +102,12 @@ private void initialize() {
         gbc.gridy++;
         optionpanel.add(wascid, gbc);
 
+        // Include Evidence
+        evidence = new JCheckBox();
+        evidence.setText(Constant.messages.getString("customreport.alertdetails.evidence"));
```

Comment:
`customreport.alertdetails.evidence = Evidence` will need be to added to the [add-on's Messages.properties file](https://github.com/zaproxy/zap-extensions/blob/master/addOns/customreport/src/main/resources/org/zaproxy/zap/extension/customreport/resources/Messages.properties). (Line 34 I believe)

Reply:
done